File: app/src/main/java/cz/kutner/comicsdb/connector/service/ComicsService.java
package cz.kutner.comicsdb.connector.service;

import cz.kutner.comicsdb.model.Comics;
import retrofit.http.GET;
import retrofit.http.Query;
import rx.Observable;

public interface ComicsService {
    @GET("/comics.php")
    Observable<Comics> getComics(@Query("id") int comicsId);
}

File: app/src/main/java/cz/kutner/comicsdb/connector/service/ComicsService.kt
package cz.kutner.comicsdb.connector.service

import cz.kutner.comicsdb.model.Comics
import retrofit.http.GET
import retrofit.http.Query

interface ComicsService {
    @GET("/comics.php")
    fun getComics(@Query("id") comicsId: Int): Comics
}
